### ISSUE TITLE
Why dapr run command cann't make app to run in background?

### ISSUE THREAD

**Issue by AbbetWang** [dapr/dapr#3772]
## In what area(s)?

dapr run command

## Ask your question here
Why dapr run command cann't make app to run in background?
It's convenient that dapr runs app in background.

**Comment by daixiang0** [comment]
Could you share your cases here?

**Comment by AbbetWang** [comment]
Background:
I followed the Hello-world project of quickstart.
When I inputs "dapr run ...."  as the picture shows.
![628cf612c82d810609cff159cacdae9](https://user-images.githubusercontent.com/28219139/137440547-02b079f7-f98f-4a55-b386-672e612a7d95.png)
I got the below result.
![e8ed3241b0c67d9eabb0acb4f5b8e91](https://user-images.githubusercontent.com/28219139/137440635-7f29cc15-49cd-4e72-9c45-ca0b2047efeb.png)

The app will shutdown when I close the termainal window.

what i wanna to do:
1. After I play  [dapr run xxx} ,the dapr app can run in backgroud other than hangup the window.
2. like the docker run -d xx, -d means detach.
